doc/protocol.md

```diff
@@ -1072,6 +1072,13 @@ The following options can be specified in the "open" control message:
    you don't set this field, the actual tag will not be checked.  To
    express that you expect the file to not exist, use "-" as the tag.
 
+* "attrs": a JSON object containing file owner and group information
+           to set. Supporting the file ownership fields returned by `fsinfo`.
+ * `uid`: an integer, the uid of the file owner (`st_uid`)
+ * `owner`: a string, or an integer, the uid of the file owner (`st_uid`)
```

Comment:
These conflict. Please document which one wins, or that specifying both is an error. Same for group.

Is there some deeper reason to allow both? AFAIUI, `owner` can take an uid, so this feels redundant.